### PR TITLE
Updating API docs with OAS3 compliant Swagger yml and viewer

### PR DESCRIPTION
**Pull request by sephcoster**
- Updating the CCDB API documents with OAS 3.0 specification complaint Swagger as well as updated language regarding API plans.
- Updated with SwaggerUI Viewer code and dependencies including minified JS

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-1.9.3-p392
+2.6.5p114
```
Comment (Scotchester) [inline .ruby-version:1]: I'm not sure if this file is necessary, but if we're going to have this file, we should probably use [the version currently in use on GH Pages](https://pages.github.com/versions/).

File: static/css/style.css [hunk 2]
```diff
@@ -393,9 +404,11 @@ nav li {
 #console a.console, 
 #queries a.queries,
 #fields a.fields,
+#api-docs a.api-docs,
 #contribute a.contribute,
 #data a.data,
 #release-notes a.release-notes,
+#api-docs a.api-docs,
```
Comment (Scotchester) [inline static/css/style.css:411]: You have accidentally added this line twice.

```suggestion
```

**Review by JeffreyMFarley — APPROVED** [review]
Approved based on https://sephcoster.github.io/api/ccdb/index.html

**Review by Scotchester — APPROVED** [review]
Looks fine enough on your fork, so none of this is blocking.

**Pull request merged**